tests/TestCase/AlgoliaEngineTest.php

```diff
@@ -51,6 +51,8 @@ public function testIndexing()
         $this->assertEquals(1, $result);
         $result = $engine->count('This should not have results', $searchablePost->getIndexName());
         $this->assertEquals(0, $result);
+        $result = $engine->count('', $searchablePost->getIndexName(), ['tagFilters' => 'test']);
```

Comment:
Shouldn't we add a test that takes in consideration the `$parameters` value? Basically add test where the value of the count is not 0.

Reply:
Yeah the test sucks but it at least tests that the parameters is passed. If you run this test on the master branch it will fail. Testing this arg with count will only work with filtering parameters, and that would require a bigger refacto of the tests. Today I'm only indexing 1 post and I have no attributes to filter on.
This should definitely be improved 👍